.editorconfig

```diff
@@ -0,0 +1,13 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+indent_size = 2
```

Comment:
😒

Reply:
@robtimp - this is in keeping with the Facebook style guide.

Reply:
I know. It just bums me out

Reply:
I agree with @robtimp on this fwiw but not enough to fight it because there is not enough consensus. The example code in swift api guidelines uses two spaces but apple uses four. Most of the open source tools out there use four but not all.

Reply:
@codytwinton  @joesus It's pretty silly that in 2019 it’s still a debate and not just a setting in the IDE